sign/dilithium/dilithium.go

```diff
@@ -6,13 +6,20 @@
 // https://pq-crystals.org/dilithium/data/dilithium-specification-round2.pdf
 //
 // Each of the eight different modes of Dilithium is implemented by a
-// subpackge.  For instance, Dilithium III can be found in
+// subpackage.  For instance, Dilithium III can be found in
 //
 //  github.com/cloudflare/circl/sign/dilithium/mode3
 //
 // If your choice for mode is fixed compile-time, use the subpackages.
 // This package provides a convenient wrapper around all of the subpackages
 // so one can be chosen at runtime.
+//
+// This package also implements two hybrid modes: EdDilithium3 and EdDilithium4.
```

Comment:
This package implements only dilithium, no more than that.
I don't see the need to include the hybrid mode here.

Reply:
Let's merge this discussion with the thread above where I try to argue that it should be logically part of this package.